File: samples/JavaScript/rename-a-file.js
var fso = new ActiveXObject("Scripting.FileSystemObject");
fso.MoveFile('input.txt', 'output.txt');
fso.MoveFile('c:/input.txt', 'c:/output.txt');
fso.MoveFolder('docs', 'mydocs');
fso.MoveFolder('c:/docs', 'c:/mydocs');

File: samples/Objective-C/rename-a-file.m
NSFileManager *fm = [NSFileManager defaultManager];

// Pre-OS X 10.5
[fm movePath:@"input.txt" toPath:@"output.txt" handler:nil];
[fm movePath:@"docs" toPath:@"mydocs" handler:nil];

// OS X 10.5+
[fm moveItemAtPath:@"input.txt" toPath:@"output.txt" error:NULL];
[fm moveItemAtPath:@"docs" toPath:@"mydocs" error:NULL];
